src/main/vct/main/Main.scala

```diff
@@ -138,8 +138,11 @@ case object Main extends LazyLogging {
       val thisThread = Thread.currentThread()
       Thread.getAllStackTraces.keySet().stream().filter(_ != thisThread)
         .filter(!_.isDaemon).forEach { thread =>
-          logger.warn(s"Non-daemon thread ${thread.getThreadGroup
-              .getName}.${thread.getName} (#${thread.getId}) is still running")
+          val tg = thread.getThreadGroup
+          logger.warn(s"Non-daemon thread ${if (tg == null)
+              "UnknownThreadGroup"
+            else
+              tg.getName}.${thread.getName} (#${thread.getId}) is still running")
```

Comment:
Cheers, but please use `Option(tg)` instead (which will be inhabited only if `tg ne null`) like e.g. `Option(tg).map(_.getName).getOrElse("UnknownThreadGroup")`.

Reply:
> Cheers, but please use `Option(tg)` instead (which will be inhabited only if `tg ne null`) like e.g. `Option(tg).map(_.getName).getOrElse("UnknownThreadGroup")`.

Like this?